src/Servers/Kestrel/Transport.Sockets/src/SocketConnectionListener.cs

```diff
@@ -23,7 +23,6 @@ internal sealed class SocketConnectionListener : IConnectionListener
         private Socket? _listenSocket;
         private int _settingsIndex;
         private readonly SocketTransportOptions _options;
-        private SafeSocketHandle? _socketHandle;
```

Comment:
I think it's fine to remove this since we probably shouldn't have taken ownership of the handle in the first place, but I want to point out to other reviewers this is a behavioral change.